.github/workflows/checks.yml

```diff
@@ -39,4 +39,4 @@ jobs:
           (cd example-1 && npx featurevisor test)
 
       - name: Run Featurevisor project tests against Go SDK
-        run: go run ./cli/main.go test --projectDirectoryPath=./example-1
+        run: go run ./cmd/main.go test --projectDirectoryPath=./example-1
```

Comment:
Slight clarification on what I said in Reddit. Typically for commands (eg. `package main`) you would have them in a directory that is exactly what you want the final executable name to be. This allows `go install` to work without any extra steps. For example, if you wanted your executable binary to be called "featurevisor", you'd put it in `cmd/featurevisor/main.go`. Then when people do `go install github.com/featurevisor/featurevisor-go/cmd/featurevisor` they'll get a binary called `featurevisor`. With your current directory structure people do `go install github.com/featurevisor/featurevisor-go/cmd` they'll get a binary called `cmd`, which is probably not what you want.